lib/client-assets.php

```diff
@@ -866,11 +866,26 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	 */
 	$allowed_block_types = apply_filters( 'allowed_block_types', true );
 
+	// Remove the default media buttion.
```

Comment:
Minor: Worth extracting to a separate function? This one's becoming quite unwieldy.